src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -94,6 +97,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
      */
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        String requestUrl = request.getRequestURI();
+        String username = requestUrl.substring("/users/".length());
+
+        // gets user
+        User user = userStore.getUser(username);
+
+        // gets bio from request
+        String bio = request.getParameter("bio");
+
```

Comment:
Do you think we need to sanitize user input around here in order to prevent XSS attacks?  Using the Jsoup is a good option! 
https://sites.google.com/codeustudents.com/summer2018/reference-guides/sanitizing-user-input